### ISSUE TITLE
Typo in the uninstall option name

### ISSUE THREAD

**Issue by vassilevsky** [funbox/init-exporter#24]
```
$ sudo upstart-export --uninstall --format upstart myapp
Error while arguments parsing:
  Argument --uninstall is not supported
```

I believe that the right spelling is `uninstall` :)

**Comment by andyone** [comment]
😟

**Comment by andyone** [comment]
Fixed in [0.12.3](https://github.com/funbox/init-exporter/releases/tag/v0.12.3).
Thank you a lot!
